libbeat/kibana/client_config.go

```diff
@@ -25,12 +25,12 @@ import (
 
 // ClientConfig to connect to Kibana
 type ClientConfig struct {
-	Protocol      string            `config:"protocol" yaml:"protocol"`
-	Host          string            `config:"host" yaml:"host"`
-	Path          string            `config:"path" yaml:"path"`
-	SpaceID       string            `config:"space.id" yaml:"space.id"`
-	Username      string            `config:"username" yaml:"username"`
-	Password      string            `config:"password" yaml:"password"`
+	Protocol      string            `config:"protocol" yaml:"protocol,omitempty"`
+	Host          string            `config:"host" yaml:"host,omitempty"`
+	Path          string            `config:"path" yaml:"path,omitempty"`
+	SpaceID       string            `config:"space.id" yaml:"space.id,omitempty"`
+	Username      string            `config:"username" yaml:"username,omitempty"`
+	Password      string            `config:"password" yaml:"password,omitempty"`
 	TLS           *tlscommon.Config `config:"ssl" yaml:"ssl"`
 	Timeout       time.Duration     `config:"timeout" yaml:"timeout"`
 	IgnoreVersion bool
```

Comment:
@urso I do like the above, having YAML tag on the config struct, since this data exist in the _kibana_ and not in the _management_ package. I could create a custom type in the management package, but we that would put us at risk to diverge.


Something to keep in mind if we ever work on the schema is to make the struct serializable.